pkg/domain/podman/update_v1_6_0_test.go

```diff
@@ -36,27 +36,27 @@ func TestSkupperNetworkStatusVolumeUpdate(t *testing.T) {
 		{
 			name:       "newer-version",
 			curVersion: "1.5.3",
-			version:    "1.5.4",
+			version:    "1.6.0",
 			applies:    true,
 			changed:    true,
 		},
 		{
 			name:       "newer-version-volume-exists",
 			curVersion: "1.5.3",
-			version:    "1.5.4",
+			version:    "1.6.0",
 			applies:    true,
 			changed:    false,
 		},
 		{
 			name:       "older-version",
-			curVersion: "1.5.4",
+			curVersion: "1.6.0",
```

Comment:
So, the volume change was introduced in 1.6, not 1.5, and therefore both the listed version and file name are changing in this PR, is that correct?

Reply:
That's right. This change was planned to land as part of 1.5.4,
but it ended up being part of the 1.6.0.